package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -33,23 +33,19 @@
 
 This module contains code to analyse protein helices using the
 HELANAL_ algorithm
-([Bansal2000]_ , [Sugeta1967]_ ).
+:cite:p:`Bansal2000` ([Sugeta1967]_ ).
 
 HELANAL_ quantifies the geometry of helices in proteins on the basis of their
 Cα atoms. It can determine local structural features such as the local
 helical twist and rise, virtual torsion angle, local helix origins and
 bending angles between successive local helix axes.
 
-.. _HELANAL: https://pubmed.ncbi.nlm.nih.gov/10798526/
+.. _HELANAL: http://www.ccrnp.ncifcrf.gov/users/kumarsan/HELANAL/helanal.html
```

Comment:
This URL is actually broken with a 404

We could link to a page on the wayback machine https://web.archive.org/web/20100818185943/http://www.ccrnp.ncifcrf.gov/users/kumarsan/HELANAL/helanal.html instead — although for right now, I am also fine with just leaving the historic broken link and then worry about link clean-up another time, especially as we don't really have a policy for what to do with broken links.

Reply:
I opted for adding the web archive link in both `references.rst` and `helix_analysis.py`. Better to fix this now while we are working on it.